PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/configuration/LivySparkBatchJobRunConfigurationType.java

```diff
@@ -48,7 +49,10 @@ public String getConfigurationTypeDescription() {
 
     @Override
     public Icon getIcon() {
-        return PluginUtil.getIcon(CommonConst.OpenSparkUIIconPath);
+        return PluginUtil.getIcon(IconPathBuilder
+                .custom(CommonConst.OpenSparkUIIconName)
+                .setPathPrefix(CommonConst.IconPathPrefix)
```

Comment:
`CommonConst.IconPathPrefix` can be the default value of `IconPathBuilder.pathPrefix`